README.md

```diff
@@ -55,21 +46,55 @@ localstack start
 
 You can build and deploy the sample application on LocalStack by running our `Makefile` commands. Run `make deploy` to create the infrastructure on LocalStack. Run `make stop` to delete the infrastructure by stopping LocalStack.
 
-## Testing the application
+Alternatively, here are instructions to deploy it manually step-by-step.
+
+### Install the dependencies
+
+We use the [Serverless Framework](https://www.serverless.com/framework/docs/getting-started/) to deploy the sample application. Install the Serverless Framework and other dependencies by running the following commands:
+
+```shell
+npm install -g serverless
+npm install
+```
+
+### Deploy the infrastructure
+
+We will use the `serverless deploy` command to deploy the infrastructure. This command will create the required resources on LocalStack. Run the following command to deploy the infrastructure:
+
+```shell
+sls deploy --stage local --verbose
+```
+
+> The `--stage` flag is specified to deploy the infrastructure to the `local` stage. It sets up Serverless to use the LocalStack plugin but only for the stage `local` to deploy the infrastructure to LocalStack only.
+
+After a few seconds, the infrastructure should be deployed successfully. You will find a `ServiceEndpoint` URL specified in the output. This URL is the endpoint for the frontend application and should look similar to: `https://vmvs1am212p.execute-api.localhost.localstack.cloud:4566/local/upload`.
+
+Next, we need to create configurations for CORS settings for our S3 bucket and SES. Run the following commands to create the configurations:
```

Comment:
We can rephrase `Next, we need to create configurations for CORS settings for our S3 bucket and SES` to `Next, we need to create configurations for CORS settings for our S3 bucket and verification for SES`